alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -227,12 +227,12 @@ func resourceAlkiraConnectorAzureVnet() *schema.Resource {
 			},
 			"size": {
 				Description: "The size of the connector, one of `SMALL`, `MEDIUM`, " +
-					"`LARGE`, `2LARGE`, `4LARGE`, `5LARGE`, `10LARGE`, `20LARGE`.",
+					"`LARGE`, `2LARGE`, `5LARGE`.",
 				Type:     schema.TypeString,
 				Required: true,
 				ValidateFunc: validation.StringInSlice([]string{
 					"SMALL", "MEDIUM", "LARGE", `2LARGE`,
-					`4LARGE`, `5LARGE`, `10LARGE`, `20LARGE`}, false),
```

Comment:
We could actually remove all those validation functions, size is always changing and API should have handled validation

Reply:
I think we have validations for size in the backend, I'll confirm and we can start removing those validations moving forward.